### PR TITLE
Upgraded libs changing test for corner case

### PR DESCRIPTION
**Pull request by jonasfj**
More upgraded libraries... Notably getting babe-runtime version 6 will mean that we don't have multiple versions of this in various apps...

### CODE REVIEW

File: test/config_test.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ suite("config", function() {
       ]
     });
 
-    assume(cfg).deep.equals(undefined);
+    assume(cfg).deep.equals({});
```
Comment (jonasfj) [inline test/config_test.js]: Lodash changed behavior for  `_.defaultsDeep`, so result changed too.. But I think this result is better if all config files is missing...

Comment (jonasfj) [inline test/config_test.js]: We could easily hack it to do the old thing... But I actually like this better.. And I doubt a major version upgrade is required to justify this...

**Comment by imbstack** [comment]
This is great except this totally needs a major version upgrade since we've decided to be serious about this now 👔

**Comment by jonasfj** [comment]
In that case I would rather find an argument for reverting the logic back to what it was before

**Comment by imbstack** [comment]
## r+
